frontend/src/v5/ui/components/shared/revisionDetails/components/revisionsListHeaderLabel/revisionsListHeaderLabel.component.tsx

```diff
@@ -21,8 +21,12 @@ import { Container } from './revisionsListHeaderLabel.styles';
 
 type IDashboardListHeaderLabel = {
 	width?: number;
+	tabletWidth?: number;
+	mobileWidth?: number;
 	className?: string;
 	children?: ReactNode;
+	hideBelowTablet?: boolean;
+	marginRight?: boolean;
 };
 
 export const RevisionsListHeaderLabel = ({ children, ...props }: IDashboardListHeaderLabel) => (
```

Comment:
I think we could reuse 

`IFixedOrGrowContainer` from  `fixedOrGrowContainer.component.tsx` but export it and rename it to `IFixedOrGrowContainerProps`

Like this

```suggestion
type IDashboardListHeaderLabel = {
	marginRight?: boolean;
} & IFixedOrGrowContainerProps;

export const RevisionsListHeaderLabel = ({ children, ...props }: IDashboardListHeaderLabel) => (
```